pkg/sql/catalog/lease/lease_test.go

```diff
@@ -980,15 +980,23 @@ CREATE TABLE t.foo (v INT);
 func TestTxnObeysTableModificationTime(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
-	skip.WithIssue(t, 85876)
 	params := createTestServerParams()
+	params.ScanMaxIdleTime = time.Millisecond
 	s, sqlDB, kvDB := serverutils.StartServer(t, params)
 	defer s.Stopper().Stop(context.Background())
 
 	// Disable strict GC TTL enforcement because we're going to shove a zero-value
 	// TTL into the system with AddImmediateGCZoneConfig.
 	defer sqltestutils.DisableGCTTLStrictEnforcement(t, sqlDB)()
 
+	_, err := sqlDB.Exec(`SET CLUSTER SETTING sql.gc_job.wait_for_gc.interval = '1s';`)
```

Comment:
Do you think it is worth encapsulating setting these 2? I see they are set together in many places and it is not obvious why.

Reply:
yes or no IMO. Yes that they both control how soon deleted data can be GC'ed, No that there is also something else controlling it for example the server param above and zone config...I don't have strong opinion on this..